app/src/ui/lib/button.tsx

```diff
@@ -27,6 +27,12 @@ export interface IButtonProps {
    */
   readonly onMouseEnter?: (event: React.MouseEvent<HTMLButtonElement>) => void
 
+  /**
+   * A function that's called when the user moves over the button with
+   * a pointer device.
+   */
```

Comment:
Something like this…?

```suggestion
  /** Called on key down. */
```